bokehjs/src/coffee/common/plot.coffee

```diff
@@ -95,6 +95,8 @@ class PlotView extends ContinuumView
         el: @$(toolbar_selector)
       })
 
+      @mget('callback')?.execute(@model)
```

Comment:
Oooooh.. right. It wasn't supposed to be inside the `if toolbar_location?` but in tail of the plot initialization (probably nee to change my editor colors :-) ). I placed the CB there because I'd like to hook the moment the plot is created so I can customize some page/app widgets in loading time.. Maybe we can add a central CB to be called on overall document init time instead of the plot init time.